flycheck.el

```diff
@@ -4170,7 +4193,14 @@ non-nil."
     (setq flycheck-display-error-at-point-timer
           (run-at-time flycheck-display-errors-delay nil 'flycheck-display-error-at-point))))
 
-
```

Comment:
This line shouldn't get lost.  I think it's one of the page breaks that we use these to separate sections in `flycheck.el`—try `C-x [` or `C-x ]` to navigate them and install `page-break-lines` to make them looker nicer than `^L`.